src/test/ui/parser/issue-80134.rs

```diff
@@ -0,0 +1,10 @@
+fn main() {
```

Comment:
Could you rename this test file to be more descriptive?

Reply:
What the name do you prefer?

Reply:
`multibyte-char-use-seperator-issue-80134.rs`? It's long but I'm trying to avoid landing more tests that are named `issue-XXXXX.rs`.